R/loginToDATIM.R

```diff
@@ -141,18 +141,18 @@ loginToDATIM <- function(config_path = NULL,
                          d2_session_name = "d2_default_session",
                          d2_session_envir = parent.frame()) {
 
-  if((!(is.null(username)) && is.null(password)) || (is.null(username) && !(is.null(password)))){
+  if ((!(is.null(username)) && is.null(password)) || (is.null(username) && !(is.null(password)))) {
```

Comment:
Seems like this could be simplified using `xor`:
```suggestion
  if (xor(is.null(username), is.null(password))) {
```